frontend/src/components/RecommendedRecipes.css

```diff
@@ -45,3 +45,8 @@ limitations under the License. */
   margin: auto;
   display: block;
 }
+.spinner-div {
+  text-align: center;
+  margin:auto;
+  padding-top: 200px;
+}
```

Comment:
Add newline at the end of each file

Reply:
GitHub puts this (-) icon on the file if there's no newline
![image](https://user-images.githubusercontent.com/35656603/90386570-aa8aa180-e07c-11ea-8a61-3591113887d4.png)
If a file has a newline, then I think it doesn't complain